autoload/thesaurus_query/backends/thesaurus_com_lookup.py

```diff
@@ -89,7 +93,10 @@ def query_cmd_handler(self, word):
                 self._loaded=True
 
         if self._backendDisabled is False:
-            self.query_result = self._Word(word)
+            try:
+                self.query_result = self._Word(word)
+            except:
+                self.query_result = EmptyWord([])
```

Comment:
Hi @Manwholikespie , suggest returning `[-1,[]]` when exception happens.

In this plugin, the backend could have three types of returns:
```
   [status, [[PoS+meaning, [word_0, word_1, ...]],  [PoS+meaning, [word_0, word_1, ...]], ...]]
status:
    0: normal,  result found, list will be returned as a nested list
    1: normal, result not found, return empty list
    -1: unexpected result from query, return empty list
```
Yes, it's a bit clumsy... But could you also follow this return pattern for now?

Basically, an unexpected result will lower the backend priority in the next query; while "not found" or "found" result will retain the priority of the backend. In this case, if the thesaurus.com did something funny again, before the update of the thesaurus plugin, I believe the return should be `[-1,[]]`, because the plugin will never yield positive result before updates. After receiving this return, the plugin will try other backends first in the future queries.